index.html

```diff
@@ -20,13 +21,13 @@
             <li><a href="#contact">Contact</a></li>
         </ul>
     </nav>
+    </header>
     <!-- Headline -->
     <section class="headline">
```

Comment:
:red_circle: [REQUIRED] I noticed that you have used the mobile background image for both the desktop and mobile versions of your project. The design template uses two different images, one for each version.

I would recommend that you use the desktop background image for the desktop version of your project and the mobile background image for the mobile version. This will ensure that your project looks consistent across all devices.

Figma
![figma](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/26736582/d6c27b6b-6444-4dc7-bdbc-31512aa1ec92)

Your Design
![ur](https://github.com/Cheptoo21/Portfolio-setup-and-mobile-first/assets/26736582/0dc2e4bb-b52d-4835-a4aa-d884ce83da9f)